.changeset/smart-garlics-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+---
+
+[Improvement] Change the default validity time of the cache to 5 minutes. If you need to customize it, use the `expires_in` option.
```

Comment:
[req] This description is outdated after the changes.

Reply:
I updated this description to reflect newer changes.